source/FreeRTOS_ND.c

```diff
@@ -1009,6 +1009,10 @@
                        }
 
                        pxICMPHeader_IPv6->ucTypeOfMessage = ipICMP_PING_REPLY_IPv6;
+
+                       /* MISRA Ref 4.14.1 [The validity of values received from external sources]. */
+                       /* More details at: https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/main/MISRA.md#directive-414. */
+                       /* coverity[misra_c_2012_directive_4_14_violation] */
                        prvReturnICMP_IPv6( pxNetworkBuffer, uxICMPSize );
```

Comment:
What about the second occurrence of prvReturnICMP_IPv6

Reply:
In the second call, the argument `uxICMPSize` is a constant expression and is not read from external sources. In line 1016 even though the value is coming from external source, it is validated via `if( uxNeededSize > pxNetworkBuffer->xDataLength )` check, but coverity is failing to detect it and is marked as a false positive.